projects/fal/src/fal/api.py

```diff
@@ -738,6 +740,10 @@ def build_app(self):
             allow_methods=("*"),
             allow_origins=("*"),
         )
+        _app.add_middleware(
+            PrometheusMiddleware, prefix="http", skip_paths=["/metrics"]
```

Comment:
this probably means we need to add `starlette_exporter` to the function requirements too https://github.com/fal-ai/fal/blob/5e95ddbe9d2d46d9826555436e56b81e1ecc7ec7/projects/fal/src/fal/api.py#L110-L111

Reply:
oh yes exactly! I wanted to ask you where to find that!